packages/mobile/src/pincode/authentication.ts

```diff
@@ -252,6 +267,43 @@ export function* getPasswordSaga(account: string, withVerification?: boolean, st
 
 type PinCallback = (pin: string) => void
 
+export async function setPincodeWithBiometrics() {
+  let pin = getCachedPin(DEFAULT_CACHE_ACCOUNT)
+  if (!pin) {
+    pin = await requestPincodeInput(true, true)
+  }
```

Comment:
i'm not sure what you mean here? i really wanted `setPincodeWithBiometrics` to be simple to call, because this will be called from the biometry screen + the settings screen. i didn't want those screens to know about how to fetch the pin. there is a `storePinWithBiometrics` function a few lines down that does explicitly require the pin as a param, is this what you mean?

Reply:
Ah yes I see, fine as is then 👍